src/containers/MyComplaints/dummyData/index.js

```diff
@@ -0,0 +1,55 @@
+let DATA = {
+  category: 'seguridad',
+  title: 'Abuso Animal',
+  description:
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.',
+  date: 'Ocurrido el 6 mayo, 2021',
+  location:
+    'Hacienda Los Fresnos, Av. Los Fresnos #345, Jalisco, Tlatomulco de Zúñiga',
+  author: 'Denuncia por José Támara',
+  authorDate: '7 de mayo, 2021',
+  fav: false,
+};
+
+let MyCompEmpty = [];
```

Comment:
The variable's name should be complete in order to give a clear idea about what the function does

Example:
```suggestion
let MyCompanyEmpty = [];
```

Reply:
Fixed!